.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug
+about: Create a bug report to help us improve
+title: '[BUG]'
+labels: kind/bug
+assignees: ''
+
+---
+
+ **Describe the bug**
+A clear and concise description of what the bug is.
+
+ **To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2.
+3.
+
+ **Expected behavior**
+A clear and concise description of what you expected to happen.
+
+ **Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+ **Knative release version**
+
+ **Additional context**
```

Comment:
You might want to add [a commented block like serving has](https://raw.githubusercontent.com/knative/serving/master/.github/issue-template.md) if we want to recommend any prow commands.

```
<!--
Pro-tip: You can leave this block commented, and it still works!

Select the appropriate areas for your issue:

/area API
/area autoscale
/area build
/area monitoring
/area networking
/area test-and-release

Classify what kind of issue this is:

/kind question
/kind bug
/kind cleanup
/kind doc
/kind feature
/kind good-first-issue
/kind process
/kind spec
/kind proposal

-->
```

Reply:
For example, you could also have
```markdown
<!--
/kind bug
-->
```

If you wanted to suggest prow commands rather/in addition to having labels auto-set.

Reply:
Lets do this later. Right now we want to get these templates in so that folks can start submitting proposals for 0.6. I would keep it simple and add these later when we have well defined areas.